src/main/webapp/dashboard.js

```diff
@@ -133,7 +156,8 @@ function addProjectTag(project, projectDiv) {
  */
 function createLocation(location) {
```

Comment:
Q: Would functions like this, which are called by createAboutMe and use the 'document' object, require 'document' to be passed in as a param?

Reply:
for calls to ```createElement,``` it should be fine. It only really matters if they are adding to the ```DOM``` i think